lib/resources/r4/charge_item_create.yaml

```diff
@@ -0,0 +1,425 @@
+                    }
+                  ],
+                  "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/CodeSystem/BillCodes-MODIFIER",
+                  "code": "26"
+                }
+              ]
+            },
+            "url": "code"
+          },
+          {
+            "valueUnsignedInt": 1,
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
+          }
+        ],
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/modifier-code"
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the MODIFIER proprietary Bill Codes Types system.</li>
+      </ul>
+
+  - name: revenue-code
+    required: 'No'
+    type: Extension
+    description: The type of revenue or cost center providing the product and/or service.
+    action: create
+    example: |
+      {
+        "valueCodeableConcept": {
+          "coding": [
+            {
+              "extension": [
+                {
+                  "valueCoding": {
+                    "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/14002",
+                    "code": "615217"
+                  },
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                }
+              ],
+              "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/CodeSystem/BillCodes-REVENUE",
+              "code": "0100"
+            }
+          ]
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/revenue-code"
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the REVENUE proprietary Bill Codes Types system.</li>
+      </ul>
+
+  - name: performing-location
+    required: 'No'
+    type: Extension
+    description: A location where the charge was performed.
+    action: create
+    example: |
+      {
+        "valueReference": {
+          "reference": "Location/683731",
+          "display": "1 West SA2"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location"
+      }
+
+  - name: custom-attribute
+    required: 'No'
+    type: Extension
+    description: A client defined custom attribute for the charge.
+    action: create
+    example: |
+      {
+        "extension": [
+          {
+            "id": "68985657",
+            "valueString": "Family Planning Indicator",
+            "url": "custom-attribute-name"
+          },
+          {
+            "valueInteger": 68985657,
+            "url": "custom-attribute-value"
+          }
+        ],
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
+      }
+    note: >
+      <ul>
+        <li>Custom-attribute-value and Custom-attribute-value are both required.</li>
+        <li>Custom-attribute-value can be of type integer, string, decimal, or date.</li>
```

Comment:
date is going to end up changing to dateTime

Reply:
[Fixed here](https://github.com/cerner/fhir.cerner.com/pull/778/files#diff-2830ed2523973e82037a23daf274840e05a6c6e05b9f698fbd5f3e7fc24f57bbR403)

Reply:
Let's go ahead and remove date/dateTime for now. As we have an issue that doesn't allow us to support that at this time.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/778/files#diff-2830ed2523973e82037a23daf274840e05a6c6e05b9f698fbd5f3e7fc24f57bbR292-R294